code/datums/spells/inflict_handler.dm

```diff
@@ -28,9 +28,11 @@
 			if("gib_brain")
 				if(ishuman(target))
 					var/mob/living/carbon/C = target
-					if(C.brain_op_stage != 4) // Their brain is already taken out
-						var/obj/item/organ/brain/B = new(C.loc)
+					var/obj/item/organ/internal/brain/B = C.get_int_organ(/obj/item/organ/internal/brain)//do we has brains?
+					if(B)
+						B.loc = get_turf(C)
 						B.transfer_identity(C)
+						C.internal_organs -= B

```

Comment:
actually the part for the inflict hanlder is still there on main, should i remove anyway? https://github.com/ParadiseSS13/Paradise/blob/master/code/datums/spells/inflict_handler.dm#L31